xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -1461,6 +1461,20 @@ CDemuxStream* CDVDDemuxFFmpeg::AddStream(int streamIdx)
     stream->flags = (CDemuxStream::EFlags)pStream->disposition;
 
     AVDictionaryEntry *langTag = av_dict_get(pStream->metadata, "language", NULL, 0);
+    if (!langTag)
+    {
+      if ((strcmp(m_pFormatContext->iformat->name, "avi") == 0) && (streamIdx > 0 && streamIdx < 10))
```

Comment:
> which correspond to the streams 1 to 9

please proof this
I don't think so. ffmpeg numbers the streams in order of occurrence.